mesonbuild/compilers/compilers.py

```diff
@@ -302,6 +304,14 @@ def sanitizer_link_args(value):
     args = ['-fsanitize=' + value]
     return args
 
+def option_enabled(boptions, options, option):
+    try:
+        if option not in boptions:
+            return False
+        return options[option].value
+    except:
```

Comment:
__[Flake8]__

[E722] do not use bare except'


<!-- Comment by [SideCI](https://sideci.com) -->